### PR TITLE
[Commands] Redirect output to stderr when using swift-run

### PR DESCRIPTION
**Pull request by aciidb0mb3r**

### CODE REVIEW

File: Sources/Commands/SwiftTool.swift [hunk 1]
```diff
@@ -448,6 +483,12 @@ public class SwiftTool<Options: ToolOptions> {
         fatalError("Must be implemented by subclasses")
     }
 
+    func redirectStdoutToStderr() {
+        self.shouldRedirectStdoutToStderr = true
+        self.stdoutStream = Basic.stderrStream
+        DiagnosticsEngineHandler.default.stdoutStream = Basic.stdoutStream
```
Comment (aciidb0mb3r) [inline Sources/Commands/SwiftTool.swift]: @hartbit Do you have suggestions to make this cleaner?

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

File: Sources/Commands/SwiftTool.swift [hunk 2]
```diff
@@ -77,39 +84,52 @@ private class ToolWorkspaceDelegate: WorkspaceDelegate {
...
     func removing(repository: String) {
-        print("Removing \(repository)")
+        stdoutStream <<< "Removing \(repository)"
+        stdoutStream <<< "\n"
+        stdoutStream.flush()
     }
 
     func warning(message: String) {
-        print("warning: " + message)
+        // FIXME: We should emit warnings through the diagnostic engine.
```
Comment (hartbit) [inline Sources/Commands/SwiftTool.swift]: Is there an issue for this? It would make for a nice starter-bug.

**Pull request merged**